### DISCUSSION TITLE
Run on launch and run on all or set of active sessions?

### DISCUSSION THREAD

**Discussion by kernelzeroday** [BishopFox/sliver#554] [General]
Is there any way to run a script of shell commands on implant connect? 

Also is there a way to run a command (think sh -c 'nproc;free -g;uptime') on a set of sessions, or all sessions?

Am i missing a documented way to do this or is this something that is possible currently?

**Comment by moloch--** [comment]
Yes, see the `reactions` command help, or for more complex automations we recommend using [`sliver-py`](https://sliverpy.readthedocs.io/en/latest/getting-started.html#automatically-interact-with-new-sessions-async)

**Comment by kernelzeroday** [comment]
awesome thank you!
